### PR TITLE
[DOC] add STVM EP docs

### PR DESCRIPTION
**Pull request by vvchernov**
Add docs for new execution provider: Standalone TVM (STVM) EP

### CODE REVIEW

File: docs/execution-providers/STVM-ExecutionProvider.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+### Build and configure TVM
+
+Install the minimal pre-requisites on Ubuntu/Debian like linux operating systems:
+```bash
+apt-get install -y python3 python3-dev python3-pip python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev llvm-12
+pip3 install numpy decorator attrs
+```
+
+Clone this repo using the `--recursive` flag to pull all associated dependencies
```
Comment (natke) [inline docs/execution-providers/STVM-ExecutionProvider.md:35]: We should say which repo: assume https://github.com/microsoft/onnxruntime

File: docs/execution-providers/index.md [hunk 2]
```diff
@@ -29,9 +29,10 @@ ONNX Runtime supports many different execution providers today. Some of the EPs
 ---|---|---|---
 |Default CPU|[NVIDIA CUDA](../execution-providers/CUDA-ExecutionProvider.md)|[Intel OpenVINO](../execution-providers/OpenVINO-ExecutionProvider.md)|[NUPHAR (tvm/llvm) ](../execution-providers/Nuphar-ExecutionProvider.md) (*preview*)|
 |[Intel DNNL](../execution-providers/oneDNN-ExecutionProvider.md)|[NVIDIA TensorRT](../execution-providers/TensorRT-ExecutionProvider.md)|[ARM Compute Library](../execution-providers/ACL-ExecutionProvider.md) (*preview*)|[Rockchip NPU](../execution-providers/RKNPU-ExecutionProvider.md) (*preview*)|
-||[DirectML](../execution-providers/DirectML-ExecutionProvider.md)|[Android Neural Networks API](../execution-providers/NNAPI-ExecutionProvider.md)|[Xilinx Vitis-AI](../execution-providers/Vitis-AI-ExecutionProvider.md) (*preview*)|
+|[Standalone TVM](../execution-providers/STVM-ExecutionProvider.md) (*preview*)|[DirectML](../execution-providers/DirectML-ExecutionProvider.md)|[Android Neural Networks API](../execution-providers/NNAPI-ExecutionProvider.md)|[Xilinx Vitis-AI](../execution-providers/Vitis-AI-ExecutionProvider.md) (*preview*)|
```
Comment (natke) [inline docs/execution-providers/index.md:32]: Something looks a bit funky with these links

File: docs/execution-providers/STVM-ExecutionProvider.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+STVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
+STVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Build
```
Comment (natke) [inline docs/execution-providers/STVM-ExecutionProvider.md:21]: The current pattern is to have the build instructions for EPs on the build page. I think it's fine for them to be here. We can decide later which is the better pattern.

**Review by natke — APPROVED** [review]

File: docs/execution-providers/index.md (hunk 2 above)
Comment (tmoreau89) [reply docs/execution-providers/index.md:32]: So the idea here is that we're positioning Standalone TVM EP as an EP for CPU, GPU and Edge.

File: docs/execution-providers/STVM-ExecutionProvider.md (hunk 3 above)
Comment (tmoreau89) [reply docs/execution-providers/STVM-ExecutionProvider.md:21]: We can definitely work on a follow up PR that moves the build instructions into the build page to match the other EP guidelines.

File: docs/execution-providers/STVM-ExecutionProvider.md (hunk 1 above)
Comment (tmoreau89) [reply docs/execution-providers/STVM-ExecutionProvider.md:35]: That's correct we'll make the correction, thank you

File: docs/execution-providers/index.md (hunk 2 above)
Comment (natke) [reply docs/execution-providers/index.md:32]: Ah ok got it, thanks

Comment (natke) [reply docs/execution-providers/index.md:32]: I was misreading the table

Comment (tmoreau89) [reply docs/execution-providers/index.md:32]: That'll look like this:
Screen Shot 2021-12-16 at 8 27 17 PM
﻿

File: docs/execution-providers/STVM-ExecutionProvider.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
+---
+title: Standalone TVM (STVM)
+description: Instructions to execute ONNX Runtime with the Standalone TVM (STVM) execution provider
+parent: Execution Providers
+nav_order: 15
+redirect_from: /docs/reference/execution-providers/STVM-ExecutionProvider
```
Comment (faxu) [inline docs/execution-providers/STVM-ExecutionProvider.md:6]: Don't think this redirect is needed since this is new, right?

Comment (tmoreau89) [reply docs/execution-providers/STVM-ExecutionProvider.md:6]: Ah I think we blindly applied the template from other EP markdown pages. Are these lines not necessary for new EP documentation pages?

Comment (natke) [reply docs/execution-providers/STVM-ExecutionProvider.md:6]: Not necessary - these were from old EP pages. It won't actually cause any harm but best to remove it

**Review by natke — APPROVED** [review]

File: docs/execution-providers/STVM-ExecutionProvider.md (hunk 4 above)
Comment (tmoreau89) [reply docs/execution-providers/STVM-ExecutionProvider.md]: I removed them to clean it up

File: docs/execution-providers/STVM-ExecutionProvider.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
+# Standalone TVM (STVM) Execution Provider
```
Comment (natke) [inline docs/execution-providers/STVM-ExecutionProvider.md:1]: Super sorry, I wasn't clear in my previous comment. The markdown header *is* required. Just not the redirect_from line,

**Review by natke — APPROVED** [review]

File: docs/execution-providers/STVM-ExecutionProvider.md (hunk 5 above)
Comment (natke) [reply docs/execution-providers/STVM-ExecutionProvider.md:1]: I'll merge it, and fix it up in a subsequent PR

**Pull request merged**